scripts/fast-reboot-dump.py

```diff
@@ -39,7 +40,7 @@ def generate_neighbor_entries(filename, all_available_macs):
         arp_output.append(obj)
 
         ip_addr = key.split(':')[2]
-        if ipaddress.ip_interface(ip_addr).ip.version != 4:
+        if ipaddress.ip_interface(str(ip_addr)).ip.version != 4:
```

Comment:
Would this fix [issue:5580](https://github.com/Azure/sonic-buildimage/issues/5580)? One comment is that: can you make this Python3 friendly by checking the Python version?

Reply:
FYI, I have a draft PR where I am working on adding support for Python 3. You can look at this PR to see if it will play nice: https://github.com/Azure/sonic-utilities/pull/1128

Reply:
> Would this fix [issue:5580](https://github.com/Azure/sonic-buildimage/issues/5580)? One comment is that: can you make this Python3 friendly by checking the Python version?


Yes. The update is friendly to both python 2 and python 3. I have verified on both python version.

Reply:
Thanks @bingwang-ms Can you please have a look at ipv6 splitting function below? it does not look that this would work with longer prefixes. Could be in a separate PR.

Reply:
Sure. Do you mean this line?
https://github.com/Azure/sonic-utilities/blob/011117bebcf05f6ef17a3d63155dec630ae88cc3/scripts/fast-reboot-dump.py#L45
What's the problem?

Reply:
nevermind. it is splitting the key and recover the Ipv6 from the key and not the ipaddr. it was an oversight.

Reply:
> Would this fix [issue:5580](https://github.com/Azure/sonic-buildimage/issues/5580)? One comment is that: can you make this Python3 friendly by checking the Python version?

Sorry for my last unclear reply. The issue https://github.com/Azure/sonic-buildimage/issues/5580 is not fixed by this PR. The unicode issue is not found in fast-reboot-dump.py in 201911 branch.